docs/03-02-runtime-components.md

```diff
@@ -1,19 +1,21 @@
-# Runtime components
+# SAP BTP, Kyma runtime components
 
-Kyma Environment Broker (KEB) serves the functionality of composing the list of components that are installed in a Runtime. The diagram and steps describe the KEB workflow in terms of calculating and processing Runtime components:
+> **NOTE:** Once all Kyma components become independent modules, KEB will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
+
+Kyma Environment Broker (KEB) serves the functionality of composing the list of components that are installed in a SAP BTP, Kyma runtime. The diagram and steps describe the KEB workflow in terms of calculating and processing Kyma runtime components:
 
 ![runtime-components-architecture](./assets/runtime-components.svg)
 
-1. During KEB initialization, the broker reads two files that contain lists of components to be installed in a Runtime:  
+1. During KEB initialization, the broker reads two files that contain lists of components to be installed in a Kyma runtime:  
 
    * `kyma-installer-cluster.yaml` file with the given Kyma version
    * `managed-offering-components.yaml` file with additional managed components that are added at the end of the base components list
 
-2. The user provisions a Runtime and selects optional components that they want to install.
+2. The user provisions a Kyma runtime and selects optional components that they want to install.
 
 3. KEB composes the final list of components by removing components that were not selected by the user. It also adds the proper global and components overrides and sends the whole provisioning information to the Runtime Provisioner.
 
-There is a defined [list of the component names](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/runtime/components). Use these names in your implementation.
+There is a defined [list of the components and their names](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/runtime/components). Use these names in your implementation.
 
 ## Disabled components
```

Comment:
The link to the disable components, in line 22, is also broken. It should probably lead to https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/runtime/disabled_components.go

Reply:
Yes, that's right 🙇‍♀️